src/i18n/ko_KR.ts

```diff
@@ -0,0 +1,669 @@
+        2: 20,
+        6: 19,
+    },
+    schoolings: {
+        '소방학교': [
+            {
+                caption: '유해물질',
+                duration: '3 Days',
+                staffList: '유해물질',
+            },
+            {
+                caption: '이동 지휘',
+                duration: '5 Days',
+                staffList: '이동 지휘',
+            },
+            {
+                caption: '항공구조 및 소방 훈련',
+                duration: '3 Days',
+                staffList: 'ARFF',
+            },
+            {
+                caption: '신속 수상 구조',
+                duration: '4 Days',
+                staffList: '신속 수상 구조',
+            },
+            {
+                caption: '원양 항해',
+                duration: '5 Days',
+                staffList: '원양 항해',
+            },
+        ],
+        '경찰 학교': [
+            {
+                caption: '특수화기전술조',
+                duration: '5 Days',
+                staffList: '특수화기전술조',
+            },
+            {
+                caption: '싸이카',
+                duration: '3 Days',
+                staffList: '싸이카',
+            },
+            {
+                caption: '진압 경찰 교육',
+                duration: '2 Days',
+                staffList: '진압 경찰 교육',
+            },
+            {
+                caption: '진압경찰장비교육',
+                duration: '5 Days',
+                staffList: '진압경찰장비교육',
+            },
+            {
+                caption: '물대포 작전 훈련',
+                duration: '7 Days',
+                staffList: '물대포 작전 훈련',
+            },
+        ],
+        '응급구조학과(응급 의료 서비스)': [
+            {
+                caption: '중환자 치료 훈련',
+                duration: '5 Days',
+                staffList: '중환자 치료 훈련',
+            },
+            {
+                caption: '긴급 이송용 오토바이',
+                duration: '3 Days',
+                staffList: '긴급 이송용 오토바이',
+            },
+        ],
+    },
+    amount: 'Quantity',
+    search: 'Search',
+    mapSearch: 'Location search',
+    alliance: 'Alliance',
+    premiumNotice:
+        'This feature extends a premium feature of the game and is therefore only available for players with a Missionchief game premium account!',
+    credits: 'Credits',
+    coins: 'Coins',
+    close: 'Close',
+    fullscreen: {
+        expand: 'Activate full screen mode',
+        compress: 'Disable full screen mode',
+    },
+    hideTitle: 'Show heading | Hide heading',
+    vehicle: 'Cars | Car | Cars',
+    building: 'Buildings',
+    station: 'Stations | Station | Stations',
+    distance: 'Distance | Distances',
+    vehicleType: 'Vehicle type',
+    noOptions: 'Sorry, no matching options.',
+    fmsReal2Show: {
+        1: 1,
+        2: 2,
+        3: 3,
+        4: 4,
+        5: 5,
+        6: 6,
+        7: 7,
+        9: 9,
+    },
+    pois: [
+        '공원',
+        '호수',
+        '병원',
+        '숲',
+        '버스 정류장',
+        '전차 정류장',
+        '기차역(지역 교통)',
+        '기차역(지역 교통 및 장거리 여행역)',
+        '화물역',</select        '
```

Comment:
There is some additionaly code, accidentally.

Reply:
I modified it

Reply:
No way that `</select` is a proper POI in korea